server/src/main/kotlin/suwayomi/tachidesk/server/ServerSetup.kt

```diff
@@ -137,6 +137,12 @@ fun applicationSetup() {
 
     databaseUp()
 
+    Settings.init()
+
+    Settings.setup()
+
+    Settings.update(Settings.fromConfig(serverConfig))
```

Comment:
This seems too complicated, we need one entry point, the steps here are a blackbox to `ServerSetup`